### PR TITLE
#265: Correct handling of un-resizeable images

### PR DESCRIPTION
**Pull request by lat9**

### CODE REVIEW

File: includes/classes/bmz_image_handler.class.php [hunk 1]
```diff
@@ -92,6 +93,20 @@ public function __construct($src, $width, $height)
...
 
+        // -----
+        // PHP provides resizing support for .jpg/.jpeg, .gif and .png files.  If the submitted
+        // image's extension isn't one of those supported, there's no sense in checking any of the
+        // file-related attributes!
+        //
+        // If that case if found, set a processing flag (used by the 'handle_image' function) to indicate
+        // as such and perform a quick return.
+        //
+        $this->file_is_supported = true;
```
Comment (brittainmark) [inline includes/classes/bmz_image_handler.class.php:104]: I would have coded this
```
       $this->file_is_supported = h_image_supported($src);
        if ($this->file_is_supported === false) {
            return;
        }
```

**Review by brittainmark — APPROVED** [review]
Other than comment made re bmz_image_handler.class.php all works fine and looks great. You can choose to leave code as is if you prefer.

File: includes/classes/bmz_image_handler.class.php (hunk 1 above)
Comment (lat9) [reply includes/classes/bmz_image_handler.class.php:104]: I agree; update forthcoming.  That was left-over from where I was originally heading down a different implementation path, thanks for the catch!

Comment (brittainmark) [reply includes/classes/bmz_image_handler.class.php:104]: no probs glad to help.

Comment (lat9) [reply includes/classes/bmz_image_handler.class.php]: That update just pushed addresses this change request.

**Pull request merged**